src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -42,6 +44,7 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("user", message.getUser());
     messageEntity.setProperty("text", message.getText());
     messageEntity.setProperty("timestamp", message.getTimestamp());
+    messageEntity.setProperty("recipient", message.getRecipient()); //added by Sebrianne
```

Comment:
Same as other comment. Please remove it.
Please apply this comment to all the files.